README.md

```diff
@@ -88,7 +89,20 @@ Since tapes are only loaded on startup, any changes to a tape requires a server
 
 #### File Name
 New tapes will be created under the `path` directory with the name `unnamed-n.json5`, where `n` is the tape number.   
-Tapes can be renamed at will, for example to give some meaning to the scenario the tape represents.
+Tapes can be renamed at will, for example to give some meaning to the scenario the tape represents.  
+If a custom `tapeNameGenerator` is provided, it will be called to produce an alternate file path under `path`
+that can be based on the tape contents.  Note that the file extension `.json5` will be appended automatically.
+
+##### Example:
+```javascript
+function nameGenerator(tapeNumber, tape) {
+  // organize in folders by request method
+  // e.g. tapes/GET/unnamed-1.json5
+  //      tapes/GET/unnamed-3.json5
+  //      tapes/POST/unnamed-2.json5
```

Comment:
I just (now) realized that tapes are not loaded recursively, so this might be a bad example (as is the one in the unit test)...  I can change?  Or maybe loading tapes recursively is on the roadmap...  let me know what you think - thanks!